docs/rules/string-constant-types.md

```diff
@@ -0,0 +1,34 @@
+# Enforce string constants are explicitly typed
+# string-constant-types
+
+This rule enforces exported string constants are explicitly typed if they are in a flow annotated file.
+
+This is important when creating enums of string values, as omitting even one type reverts the type to `string` (`'foo' | 'bar' | string` -> `string`)
+
+## Examples of incorrect code for this rule:
+
+```js
+/* eslint saxo/string-constant-types: "error" */
+// @flow
+export const foo = 'foo'
+```
+
+## Examples of correct code for this rule:
+
+```js
+/* eslint saxo/string-constant-types: "error" */
+/* eslint-disable flowtype/require-valid-file-annotation */
+export const foo = 'foo'
```

Comment:
I would put this example in separate section (e.g. How to bypass the rule) - this is example of incorrect code that is suppressed

Reply:
This is supposed to be an example of the output for a non flow annotated file. Once we've turned on require-valid-file-annotation we'll add the eslint disable comments to all the existing files.

Regarding your below comment, adding `// @flow` to other examples should clear this up